src/sync/pull-requests.test.ts

```diff
@@ -332,12 +331,103 @@ describe("sync/pull-request", () => {
+				expect.anything()
+			).mockResolvedValue(6);
+
+			for (let i = 0; i < 12; i++) {
+				githubUserTokenNock(DatabaseStateCreator.GITHUB_INSTALLATION_ID);
+			}
+
+			githubNock
+				.get("/repos/integrations/test-repo-name/pulls/51").times(2)
+				.reply(200, pullRequest)
+				.get("/repos/integrations/test-repo-name/pulls/51/reviews").times(2)
+				.reply(200, reviewsPayload)
+				.get("/users/test-pull-request-reviewer-login").times(2)
+				.reply(200, {
+					login: "test-pull-request-reviewer-login",
+					avatar_url: "test-pull-request-reviewer-avatar",
+					html_url: "test-pull-request-reviewer-url",
+					email: "test-pull-request-reviewer-login@email.test"
+				})
+				.get("/users/test-pull-request-author-login").times(2)
+				.reply(200, {
+					login: "test-pull-request-author-login",
+					avatar_url: "test-pull-request-author-avatar",
+					html_url: "test-pull-request-author-url"
+				})
+				.get("/users/integrations").times(2)
+				.reply(200, {
+					login: "integrations",
+					avatar_url: "integrations-avatar",
+					html_url: "integrations-url"
+				});
+
+			const nockPage1 = githubNock
+				.get("/repos/integrations/test-repo-name/pulls?per_page=100&page=4&state=all&sort=created&direction=desc")
+				.reply(200, pullRequestList);
+
+			const nockPage2 = githubNock
+				.get("/repos/integrations/test-repo-name/pulls?per_page=100&page=5&state=all&sort=created&direction=desc")
+				.reply(200, pullRequestList);
+
+			jiraNock.post("/rest/devinfo/0.10/bulk", buildJiraPayload("1", 2)).reply(200);
+
+			await expect(processInstallation()({
+				installationId: DatabaseStateCreator.GITHUB_INSTALLATION_ID,
+				jiraHost
+			}, sentry, getLogger("test"))).toResolve();
+			expect(nockPage1.isDone()).toBeTruthy();
+			expect(nockPage2.isDone()).toBeTruthy();
+			expect((await RepoSyncState.findByPk(repoSyncState!.id)).pullCursor).toEqual("31");
+		});
+
+		it("parallel fetching when FF is over 5 should stop", async () => {
+
+			pullRequestList[0].title = "TES-15";
+
+			when(numberFlag).calledWith(
+				NumberFlags.INCREASE_BUILDS_AND_PRS_PAGE_SIZE_COEF,
+				expect.anything(),
+				expect.anything()
+			).mockResolvedValue(6);
+
+			for (let i = 0; i < 2; i++) {
+				githubUserTokenNock(DatabaseStateCreator.GITHUB_INSTALLATION_ID);
+			}
+
+			const nockPage1 = githubNock
+				.get("/repos/integrations/test-repo-name/pulls?per_page=100&page=4&state=all&sort=created&direction=desc")
+				.reply(200, []);
+
+			const nockPage2 = githubNock
+				.get("/repos/integrations/test-repo-name/pulls?per_page=100&page=5&state=all&sort=created&direction=desc")
+				.reply(200, []);
+
+			await expect(processInstallation()({
+				installationId: DatabaseStateCreator.GITHUB_INSTALLATION_ID,
+				jiraHost
+			}, sentry, getLogger("test"))).toResolve();
+			expect(nockPage1.isDone()).toBeTruthy();
+			expect(nockPage2.isDone()).toBeTruthy();
+			expect((await RepoSyncState.findByPk(repoSyncState!.id)).pullStatus).toEqual("complete");
```

Comment:
I don't get how the unit test assert. What are we asserting exactly ( in terms of the parallel?)